prosoar/templates/index.html

```diff
@@ -9,8 +9,10 @@
 
   <script type="text/javascript" src="js/MooTools/mootools-core.js"></script>
   <script type="text/javascript" src="js/MooTools/mootools-more.js"></script>
+  {% if get_locale() != 'en'%}
 
-  <link rel="gettext" href="LC_MESSAGES/{{ get_locale() }}.json" />
+    <link rel="gettext" href="LC_MESSAGES/{{ get_locale() }}.json" />
+  {% endif %}
 
```

Comment:
Instead of adding en.json (#19), just dont put anything here for english language, as all strings are already in english by default. Works for me&trade; :smile: